cgp/genome.py

```diff
@@ -394,10 +444,12 @@ def _replace_invalid_address_alleles(self, dna: List[int], rng: np.random.Random
                 dna[gene_idx] = gene_value
 
     def _get_addable_nodes(
-        self, node_dependencies: Dict[int, Set[int]], used_node_indices: List[int] = [],
+        self,
```

Comment:
are these changes necessary for this PR?

Reply:
No. I think there is an error in my pre-commit hooks, will change this and revert these changes